src/git-command-manager.ts

```diff
@@ -164,17 +167,14 @@ class GitCommandManager {
     return output.exitCode === 0
   }
 
-  async fetch(fetchDepth: number, refSpec: string[]): Promise<void> {
-    const args = [
-      '-c',
-      'protocol.version=2',
-      'fetch',
-      '--no-tags',
-      '--prune',
-      '--progress',
-      '--no-recurse-submodules'
-    ]
-    if (fetchDepth > 0) {
+  async fetch(refSpec: string[], fetchDepth?: number): Promise<void> {
+    const args = ['-c', 'protocol.version=2', 'fetch']
+    if (!refSpec.some(x => x === refHelper.tagsRefSpec)) {
```

Comment:
do we need to worry about casing? `Refs/Tags/*` vs `refs/tags/*`

Reply:
not worried in this case, just used to avoid `--no-tags` flag. I think would be safe to add `--no-tags` but would just look weird when tags refspec supplied. Also, customer doesn't control the refspec, it's generated from refHelper which also exports the const `tagsRefSpec`.